maker-frontend/src/MakerApp.tsx

```diff
@@ -49,12 +49,13 @@ export default function App() {
 
```

Comment:
nit: I believe you can open this PR against `master`, there's no need to point it to a different branch